prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -26,6 +27,7 @@ open class Receiver(
     val name: String,
     val organizationName: String,
     val topic: String,
+    val customerStatus: String,
```

Comment:
Let's contstrain the values for this using an enum type eg,

enum class CustomerStatus { inactive, testing, active }

Put it somewhere it can be shared between Sender.kt and Receiver.kt.  Maybe put it in SettingsProvider.kt?

Reply:
Updated